control/controller.go

```diff
@@ -33,7 +34,7 @@ type Controller struct {
 	transportServer            transport.Server
 	lsmHolder                  *LsmHolder
 	offsetsCache               *offsets.Cache
-	topicMetaManager           *topicmeta.Manager
+	TopicMetaManager           *topicmeta.Manager
```

Comment:
Why is this public?

Reply:
yeah this should not be public. Will fix this.